modules/steam.nix

```
@@ -253,34 +253,65 @@ in
         enable = mkOption {
           type = types.bool;
           default = false;
-          description = ''
+          description = lib.mdDoc ''
             Whether to enable the Steam Deck UI.
 
             When enabled, you can either launch the Steam Deck UI
             from your Display Manager or by running `steam-session`.
           '';
         };
 
+        autoStart = mkOption {
+          type = types.bool;
+          default = false;
+          description = lib.mdDoc ''
+            Whether to automatically launch the Steam Deck UI on boot.
+
+            No Display Managers may be enabled in conjunction with this option.
```

```suggestion
            Traditional Display Managers cannot be enabled when may be enabled in conjunction with this option.
```

(Mirroring my other nitpick)